### PR TITLE
chore: major changes to profile

### PR DESCRIPTION
**Pull request by krshdev**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1218"></a>

### CODE REVIEW

File: public/data/krshdev.json [hunk 1]
```diff
@@ -1,41 +1,83 @@
...
       "url": "https://twitter.com/krshdev",
       "icon": "twitter"
     },
     {
-      "name": "Checkout my website",
-      "url": "https://krshdev.co",
-      "icon": "globe"
+      "name": "Blogs & Articles about Development",
+      "url": "https://blog.krshdev.co",
+      "icon": "book
```
Comment (krshdev) [inline public/data/krshdev.json:20]:
```suggestion
      "icon": "book",
```

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

File: public/data/krshdev.json [hunk 2]
```diff
@@ -1,41 +1,83 @@
...
-      "icon": "book",
-      "color": "grey",
-      "description": "Started Learning to Code",
-      "url": "https://krshdev.co"
+      "title": "Became a Maintainer at WebXDAO",
+      "date": "October 2021",
+      "icon": "github",
+      "color": "#1CADFD",
+      "description": "Started a volunteer position at WebXDAO as a core maintainer and moderator!",
+      "url": "https://bio.link/webxdao",
```
Comment (krshdev) [inline public/data/krshdev.json:40]:
```suggestion
      "url": "https://bio.link/webxdao"
```

File: public/data/krshdev.json [hunk 3]
```diff
@@ -1,41 +1,83 @@
...
+      "description": "I got the pleasure to help new hackers, get into the universe of hackathons, through MLH",
+      "url": "https://mlh.io"
+    },
+    {
+      "title": "Became a Maintainer at EddieHub",
+      "date": "December 2021",
+      "icon": "github",
+      "color": "#1CADFD",
+      "description": "Started a volunteer position at EddieHub as a maintainer for the Hacktoberfest Practice Repository",
+      "url": "http://eddiehub.org",
```
Comment (krshdev) [inline public/data/krshdev.json:72]:
```suggestion
      "url": "http://eddiehub.org"
```

**Review by schmelto — APPROVED** [review]
## Looking good to me

### TLDR;

Thank you for the PR. Your suggested changes looking good to me.


---

### Full details

Thank you for the PR. Your suggested changes looking good to me.
Feel free to checkout other issues. 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Review by Pradumnasaraf — APPROVED** [review]
LGTM!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Efficient, Maintainable

File: public/data/krshdev.json [hunk 4]
```diff
@@ -1,41 +1,83 @@
...
+      "url": "https://blog.krshdev.co",
+      "icon": "book"
+    },
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://linkedin.com/in/krshdev",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Check out my instagram",
```
Comment (Panquesito7) [inline public/data/krshdev.json:28]: You might want to capitalize the `i` here. What do you think?
```suggestion
      "name": "Check out my Instagram",
```

**Review by emmalearnscode — APPROVED** [review]
Looks good 👍🏼

**Review by kumarsonsoff3 — APPROVED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
LGTM!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by schmelto — APPROVED** [review]
## Looking good to me

Your suggested changes looking good to me.
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Pull request merged**